mat64/io.go

```diff
@@ -0,0 +1,60 @@
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package mat64
+
+import (
+	"encoding/binary"
+	"io"
+)
+
+var (
+	littleEndian  = binary.LittleEndian
+	bigEndian     = binary.BigEndian
+	defaultEndian = littleEndian
+)
+
+func writeBin(w io.Writer, data interface{}) error {
+	return binary.Write(w, defaultEndian, data)
+}
+
+func writeInt(w io.Writer, n int) error {
+	return writeBin(w, int64(n))
+}
+
+func writeSlice(w io.Writer, data []float64) error {
+	var err error
+
```

Comment:
Actually, given that I think it's clearer with readSlice removed, writeSlice is probably better removed instead.